Sming/Arch/Host/Components/hostlib/options.h

```diff
@@ -31,6 +31,10 @@
 	XX(help, no_argument, "Show help", nullptr, nullptr, nullptr)                                                      \
 	XX(uart, required_argument, "Enable UART server", "PORT", "Which UART number to enable",                           \
 	   "e.g. --uart=0 --uart=1 enable servers for UART0, UART1\0")                                                     \
```

Comment:
There is one side effect when calling this command  `make run HOST_UART_OPTIONS="--uart=0 --device=/dev/ttyUSB0"`. On Linux a new terminal is still opened although it is not needed. Can you adjust also the `run` makefile target? If it is too much of hassle you can also leave it for another PR.

Reply:
Tools use this variable separately to easiest fix is to just set `ENABLE_HOST_UARTID=` on the command line.